AAI/FAQ.md

```diff
@@ -4,21 +4,235 @@ title: AAI FAQ
 permalink: aai-faq
 ---
 
-### Can the output of `/userinfo` be used as a JWT for Authentication?
+A collection of questions (and hopefully useful answers).
 
-The spec says that `/userinfo` may be formatted as a JWT. A Clearinghouse that
-sends an access token to a `/userinfo` endpoint might get either JSON or a
-formatted and signed JWT from a broker. The JWT can be used downstream as
-an authentication mechanism if downstream services like Clearinghouses
-support it and know what to do with it. 
+<hr style="border: 2px solid; margin: 2em auto;"/>    
 
-It is probable that a special token endpoint will exist in a future
-version of this profile that should prevent the `/userinfo` endpoint
-from being overloaded.
+## Flows
+
+The following sequence diagrams are included to help explain the intended flows
+documented in the accompanying specification. 
+
+### What is the complete end to end flow using `/userinfo`?
+
+(last updated June 2022)
+
+The flow as used by Elixir uses the initial *Passport-Scoped Access Token* as
+a token handed to downstream resource servers. These servers can use this token, in conjunction
+with a callback to the broker's *Userinfo Endpoint*, to obtain the *Passport* content in
+JSON format.
+
+{% plantuml %}
+
+hide footbox
+skinparam BoxPadding 10
+skinparam ParticipantPadding 20
+
+box "Researcher"  #eee
+actor       "User Agent"                as user
+participant Client                      as client
+end box
+
+box "AAI"
+participant Broker                      as broker
+collections "IdP"                       as idps
+end box
+
+box "Data Owner"
+collections "Visa Issuer(s)"            as issuers
+end box
+
+box "Data Holder"
+participant "Clearing House"            as clearing
+participant "Data"                      as data
+end box
+
+==OIDC==
+
+ref over user, client, broker, idps
+OIDC flow results in the client holding a *Passport-Scoped Access Token*.
+end ref
+
+==Use==
+
+client -> clearing : Client requests data
+note right
+{
+  Authorization: Bearer <Passport-Scoped Access Token>
+}
+end note
+
+clearing -> broker : Clearing house asks for passport content
+note right
+{
+  Authorization: Bearer <Passport-Scoped Access Token>
+}
+end note
+
+group#DeepSkyBlue #LightSkyBlue Informative Only (not defined in the specification)
+broker -> issuers : Fetch signed visa(s) for user
+broker <- issuers : Return signed visa(s) for user
+note right
+[
+  "<visa1>",
+  "<visa2>"
+]
+end note
+end
+
+clearing <- broker : UserInfo Endpoint returns passport content (plain JSON, not signed JWT)
+note right
+{
+  "iss": "https://<issuer-website>/",
+  "sub": "<subject-identifier>",
+  "jti": "<token-identifier>",
```

Comment:
**jti** (short for JWT ID) is not returned from UserInfo, because the response is not a JWT
```suggestion
```